packages/admin-ui/src/combobox/combobox-item.ts

```diff
@@ -1,24 +0,0 @@
-import { createComponent, useElement } from '@vtex/admin-ui-react'
-import type { ComboboxItemOptions } from 'ariakit/combobox'
-import { useComboboxItem } from 'ariakit/combobox'
-import type { HTMLProps } from 'react'
-import type { AnyObject } from '@vtex/admin-ui-util'
-
-// TODO this intermediary component is a workaround for typing conflits
```

Comment:
Tested that we now could use comboboxItem directly without type errors. So this intermediary component was removed